asv.conf.json

```
@@ -27,12 +27,12 @@
         "python -mpip install seaborn",
         "python -mpip install pygments",
         "python -mpip install {wheel_file}",
-        "python -mpip install -U qiskit-ignis==0.6.0",
+        "python -mpip install -U qiskit-experiments",
```

We need to pin this to the version so we don't get a new release and skew the benchmark data because the circuit changes:

```suggestion
        "python -mpip install -U qiskit-experiments==0.3.0",
```